test/unit/lib/addMessages.js

```diff
@@ -29,6 +31,31 @@ describe('addMessages', () => {
     expect(openServices[0].openingTimesMessage).to.be.equal('Open 24 hours');
   });
 
+  it('should return the opening times message and open state when between 12:00am and 01:00am Bristish Summer Time', () => {
+    const spanningSundayMidnightOrg = {
+      openingTimes: {
+        general: {
+          monday: [{ opens: '0:00', closes: '20:00' }, { opens: '23:00', closes: '23:59' }],
+          tuesday: [{ opens: '00:00', closes: '23:59' }],
+          wednesday: [{ opens: '00:00', closes: '23:59' }],
+          thursday: [{ opens: '00:00', closes: '23:59' }],
+          friday: [{ opens: '00:00', closes: '23:59' }],
+          saturday: [{ opens: '00:00', closes: '23:59' }],
+          sunday: [{ opens: '10:00', closes: '16:00' }, { opens: '23:00', closes: '23:59' }],
+        },
+      },
+    };
+    const pharmacies = [spanningSundayMidnightOrg];
+
+    const justAfterMidnightSundayBST = '2017-10-15T23:00:53.000Z';
+    // timezone required for correct results
+    const momentTime = moment(justAfterMidnightSundayBST).clone().tz('Europe/London');
+    const openServices = addMessages(pharmacies, momentTime);
```

Comment:
You can demonstrate that the timezone is needed by removing the '.clone().tz..' part of the line. It will then fail with the error seen on live.